util.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Install deps
+dnf -y install jq dosfstools
```

Comment:
This should not be in a util script. If these scripts are intended to be run locally as well they should not have such dnf installs in them